kong.conf.default

```diff
@@ -486,6 +486,12 @@
                                  # specified name). The format is a (case
                                  # insensitive) comma separated list.
 
+#dns_valid_ttl =                 # TTL in seconds for any valid DNS record.
+                                 # If not specified the DNS resolver will
+                                 # honour the TTL specified in the DNS record.
+                                 # If given, it will override the TTL for any
+                                 # valid record received.
```

Comment:
This description is somewhat confusing to me. How about a more concise approach, less repetitive:
```
By default, DNS records are cached using the TTL value
of a response. If this property receives a value (in seconds),
it will override the TTL for all records.
```